app/components/job/details/job-progress-status/job-progress-status.component.ts

```diff
@@ -37,7 +37,7 @@ export class JobProgressStatusComponent implements OnChanges, OnDestroy {
     constructor(private poolService: PoolService, private nodeService: NodeService) {
         this.poolData = poolService.get(null);
         this.data = nodeService.list(null, {
-            maxResults: 1000,
+            pageSize: 1000,
```

Comment:
we actually override the default to be 50 for all call, so I need to force it back